app/workers/gr_sync/notification_worker.rb

```diff
@@ -0,0 +1,8 @@
+module GrSync
+  class NotificationWorker
+    include Sidekiq::Worker
+
+    def perform(_notification)
```

Comment:
I'm working on implementing it now. I wanted to pull request what I've done so far though since the implementation of the notifications felt like a distinct thing from setting up the subscriptions.